src/components/Tooltip/Tooltip.md

```diff
@@ -5,21 +5,39 @@
 To use a placeholder in its most basic form and with all defaults: provide two properties to the component, `label` and `details`
 
 ```jsx
-import React, { useRef, useState, useEffect } from 'react'
 import styles from './TooltipExamples.module.scss'
 ;<>
   <div className={styles.basicExample}>
-    Basic Tootlip <Tooltip label="Flip" details="Hi!" />
+    Basic Tooltip <Tooltip label="Flip" details="Hi!" />
   </div>
 </>
 ```
 
+### Children
+
+If you don't want to use the default information 'i' SVG icon, you can suppy your own
+custom children and wrap those within the tooltip. You can even steal the tooltip's own
+CSS Module styles (see `tipStyles` below for example).
+
+```jsx
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import styles from './TooltipExamples.module.scss'
+import tipStyles from './Tooltip.module.scss'
+;<>
+  <div className={styles.basicExample}>
+    <Tooltip popperBoxStyles={styles.CustomTipExample} label="burger icon" details="Burgers are so tasty!" placement={'right'} >
+        <FontAwesomeIcon icon={['far', 'hamburger']} className={tipStyles.icon} />
+    </Tooltip>
+  </div>
+</>
+```
+
+
 ### `placement`
 
 Customize the **Starting** position of a Tooltip
 
 ```jsx
-import React, { useRef, useState, useEffect } from 'react'
```

Comment:
We don't need to import react and hooks the that component is already importing itself -- react-styleguidist will just pull in the component and it'd deps